src/components/AppsMenu/components/AppItem.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import get from 'lodash/get'
+
+import { models } from 'cozy-client'
+import Buttons from 'cozy-ui/transpiled/react/Buttons'
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import cx from 'classnames'
+
+import { appShape } from 'proptypes/index'
+import stack from 'lib/stack'
+
+import styles from 'styles/apps-menu.styl'
+
+const getAppDisplayName = get(models, 'applications.getAppDisplayName', app => {
```

Comment:
nit: not conventional... 🤔 maybe too tricky?

Reply:
Removed 👍